client/extensions/woocommerce/app/store-stats/store-stats-chart/index.js

```diff
@@ -90,19 +100,22 @@ class StoreStatsChart extends Component {
 				<Tabs data={ chartData }>
 					{ tabs.map( ( tab, tabIndex ) => {
 						if ( ! isLoading ) {
-							const itemChartData = this.buildChartData( data[ selectedIndex ], tabs[ tabIndex ] );
-							const delta = calculateDelta( data[ selectedIndex ], data[ selectedIndex - 1 ], tabs[ tabIndex ].attr, unit );
+							const itemChartData = this.buildChartData( orderData[ selectedIndex ], tabs[ tabIndex ] );
+							const delta = this.getDeltaByStat( tab.attr );
 							return (
 								<Tab
 									key={ tab.attr }
 									index={ tabIndex }
+									label={ tab.label }
 									selected={ tabIndex === selectedTabIndex }
 									tabClick={ this.tabClick }
 								>
-									<div>{ tab.label }</div>
-									<div>{ itemChartData.value }</div>
-									<div>{ delta.value }</div>
-									<div>{ delta.since }</div>
+									<span className="store-stats-chart__value value">{ itemChartData.value }</span>
+									<Delta
+										value={ `${ Math.abs( Math.round( delta.percentage_change * 100 ) ) }%` }
+										classNames={ `${ delta.favorable } ${ delta.direction }` }
+										suffix={ `since ${ moment( delta.reference_period ).format( 'MMM D' ) }` }
```

Comment:
This will be a function of the unit, eg when looking at years:
```js
... .format( 'YYYY' ) }
```
We normalize for each case in `parseChartData`:
https://github.com/Automattic/wp-calypso/blob/master/client/state/stats/lists/utils.js#L151-L154

Should we do the same in delta normalization?